src/circleci-base/scripts/docker-login.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+docker login --username "$(echo "${DOCKER_USER_64}" | base64 --decode)" --password "$(echo "${DOCKER_PASS_64}" | base64 --decode)"
```

Comment:
If this still complains with `--password-stdin`, then to prevent adding additional warnings to the logs this should probably be fixed.  Piping the echoed password similar to inital attempts.

Reply:
Done. It still has one last complain:

Reply:
Yes, we do need to do this, we don't want to leave user/pass stored in the file system.  

But, we this should probably be a new ticket - implement a credential helper ( https://www.passwordstore.org/ ? ) to avoid storing unencrypted credentials in the file system.